USER_MANUAL.MD

```diff
@@ -0,0 +1,5 @@
+# SpatialOS GDK for Unreal Test Gyms Project 
+
+#### How to add a gym 
+* Make a copy of the empty gym and give it a descriptive name.
+* If the gym is intended to be merged back in then consider adding a text object containing the gym name.
```

Comment:
Make this active voice as it's easier to follow.
Also, this is a public repo, and we have said we are not accepting contributions - who can merge a new gym into the repo?

```suggestion
If you are a contributor(*) to the Test Gyms Project and plan to merge your gym into the repository, add a text object containing the gym's name.

(*) Note that we are not accepting public contributions at this time.
```